main.py

```diff
@@ -1,7 +1,42 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # Create genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
Try to avoid code duplication while creating genres and actors. Prefer creating these objects in a loop while iterating over data structure with required values